docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -194,7 +194,7 @@ Actions that allow you to interact with file upload input elements.
 * [Populate File Upload Input](../../reference/test-api/testcontroller/setfilestoupload.md)
 * [Clear File Upload Input](../../reference/test-api/testcontroller/clearupload.md)
 
-> The file upload actions only allow you to manage the list of files you want to upload. These files are uploaded to the server after you initiate upload, for example, when you [click](../../reference/test-api/testcontroller/click.md) the **Upload** or **Submit** button on a page.
+> The file upload actions allow you to manage the list of files you want to upload. The server receives files after you begin upload, for example, when you [click](../../reference/test-api/testcontroller/click.md) the **Upload** or **Submit** button on a page.
```

Comment:
`The server receives files after you begin upload, for example, when you [click](../../reference/test-api/testcontroller/click.md) the **Upload** or **Submit** button on a page.`

Can you clarify what you're trying to say here? For example, do you mean:

`For example, when you [click](../../reference/test-api/testcontroller/click.md) the **Upload** or **Submit** button on a page, the files are uploaded to the server.`

Reply:
I made an attempt to avoid passive voice here. Apparently, a poor one. There was no intent to change the meaning of this sentence.

Reply:
Even though "are uploaded to the server" is passive, I think the passive voice is fine in this instance.